.eslintrc.json

```diff
@@ -48,6 +51,10 @@
     "react/jsx-tag-spacing": ["warn"],
     "react/self-closing-comp": "warn",
     "space-in-parens": ["warn", "never"],
-    "space-infix-ops": ["warn"]
+    "space-infix-ops": ["warn"],
+    "no-var": ["error"],
+    "no-undef": ["error"],
+    "no-unused-vars": ["error"],
```

Comment:
Can the linter automatically fix `no-unused-vars`? Same question for the other config options here. Our decision currently is that linting issues shouldn't block a commit or PR.